src/app/public/testing/infinite-scroll/infinite-scroll-fixture.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  DebugElement
+} from '@angular/core';
+import {ComponentFixture} from '@angular/core/testing';
+import {SkyAppTestUtility} from '@skyux-sdk/testing';
+
+/**
+ * Provides information for and interaction with a SKY UX infinite scroll component.
+ * By using the fixture API, a test insulates itself against updates to the internals
+ * of a component, such as changing its DOM structure.
+ */
+export class SkyInfiniteScrollFixture {
+  private debugElement: DebugElement;
+  private skyBtnSelector = 'div.sky-infinite-scroll > button.sky-btn';
+
+  constructor(
+    private fixture: ComponentFixture<any>,
+    skyTestId: string
+  ) {
+    this.debugElement = SkyAppTestUtility.getDebugElementByTestId(fixture, skyTestId, 'sky-infinite-scroll');
+
+    this.waitForComponentInitialization();
+  }
+
+  public isEnabled(): boolean {
+    return !!this.debugElement.componentInstance.enabled;
+  }
+
+  public isButtonAvailable() {
```

Comment:
Thinking about naming conventions here... I see we name a [similar fixture pattern](https://github.com/blackbaud/skyux-popovers/blob/master/src/app/public/testing/dropdown/dropdown-fixture.ts#L80) `dropdownMenuIsVisible()`, where we treat it as public propertie with a getter only. Should we stay with that approach and do something like `public get loadMoreButtonIsVisible(): boolean` ? 
Let me know if you see other examples out there - this was just one I came up with quickly.